src/sign-up/dto/sing-up-response.dto.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { SignUpResponse } from "./sign-up-response.dto";
+
+describe("SignUpResponse", () => {
+  it("should create a LoginResponse object with optional properties", () => {
+    const loginResponse = new SignUpResponse();
```

Comment:
If we are going to write tests to models, it should fail when we add a new property. so add property to test

Reply:
we cannot verify the properties within the dto.

Reply:
we should not write unit tests only for coverage. it should also secure future developments